pkg/cache/types/types.go

```diff
@@ -35,8 +35,8 @@ type ResponseType int
 const (
 	Endpoint ResponseType = iota
 	Cluster
-	Route
 	ScopedRoute
+	Route
```

Comment:
this is nitpicky but could this technically be a breaking change? in case someone had comparisons on this type, seems unfortunate but maybe